main/tests.py

```diff
@@ -69,3 +69,16 @@ def test_unlogged_user_access_to_homepage(self, client):
         response = client.get("/homepage/")
         assert response.status_code == 302
         assert response['Location'] == '/login/?next=/homepage/'
+
+
+@pytest.mark.django_db
+class TestOrdersView:
```

Comment:
maybe need to add a test that shows we are displaying only the relevant orders in HTML